.github/workflows/charm4py.yml

```diff
@@ -0,0 +1,34 @@
+name: Charm4py
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: build-charm++
+      run: ./build charm4py netlrts-linux-x86_64 tcp -g -j4 --with-production
+    - name: build-charm4py
+      run: |
+        cd ..
+        git clone git clone https://github.com/UIUC-PPL/charm4py
+        export PYTHONPATH="$PWD/charm4py"
+        
+        mkdir -p charm4py/charm_src/charm/bin && cp bin/charmrun charm4py/charm_src/charm/bin
+        mkdir -p charm4py/charm_src/charm/lib && cp lib/libcharm.so charm4py/charm_src/charm/lib
+        mkdir -p charm4py/charm_src/charm/include && cp -r -L include/* charm4py/charm_src/charm/include
+        
+        cd charm4py
+        export CHARM4PY_BUILD_CFFI=1
+        python3 setup.py build_ext --inplace
+        python2 setup.py build_ext --inplace
+        
+        python3 auto_test.py
```

Comment:
This line duplicates the test step below?

Reply:
Should be fixed now.